lib/tasks/emojis.rake

```diff
@@ -55,4 +95,11 @@ namespace :emojis do
     File.write(dest, Oj.dump(map))
     puts "Wrote emojo to destination! (#{dest})"
   end
+
+  desc 'Generate emoji variants with white borders'
+  task :generate_borders, [] do |task, args|
+    for cc in args.extras
+      gen_border cc
+    end
+  end
```

Comment:
My idea was that the unicode characters to generate files from would be hardcoded in here.